src/components/createRoute/index.jsx

```diff
@@ -0,0 +1,813 @@
+                            zoom={12}
+                            mapContainerStyle={{ width: '100%', height: '100%' }}
+                            options={{
+                                zoomControl: true,
+                                streetViewControl: false,
+                                mapTypeControl: true,
+                                fullscreenControl: true
+                            }}
+                            onLoad={map => setMap(map)}
+                        >
+                            <Marker position={center} />
+                            {directionResponse && (
+                                <DirectionsRenderer directions={directionResponse} />
+                            )}
+                        </GoogleMap>
+                    </div>
+
+                    <Form.Item
+                        label="Enter the origin's address: "
+                        name="originAddress"
+                        rules={[
+                            InputRules.required(
+                                inputValidationErrorMessages.EMPTY_ORIGIN_ADDRESS
+                            )
+                        ]}
+                    >
+                        <PlacesAutocomplete
+                            value={originAddress}
+                            onSelect={selectOriginAddressAsync}
+                            onChange={changeOriginAddress}
+                        >
+                            {({
+                                getInputProps,
+                                suggestions,
+                                getSuggestionItemProps,
+                                loading
+                            }) => (
+                                <div>
+                                    <Input
+                                        {...getInputProps({
+                                            placeholder: 'Origin address'
+                                        })}
+                                    />
+
+                                    <div className="description-list">
+                                        {loading ? <div>Loading...</div> : null}
+
+                                        {suggestions.map((suggestion) => {
+                                            return (
+                                                <div
+                                                    {...getSuggestionItemProps(suggestion)}
+                                                    className="description-item"
+                                                >
+                                                    {suggestion.description}
+                                                </div>
+                                            );
+                                        })}
+                                    </div>
+                                </div>
+                            )}
+                        </PlacesAutocomplete>
```

Comment:
I think you can create a separate component with PlacesAutocomplete and others components within it